.circleci/config.yml

```diff
@@ -2,11 +2,10 @@ version: 2.1
 
 references:
   npm_cache_key: &npm_cache_key
-                   v1-dependency-npm1-{{ checksum "package-lock.json" }}
+                   v1-dependency-npm1-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
```

Comment:
I add `package.json`'s checksum to prevent restoring the wrong cache if someone updated `package.json` and didm't commit `package-lock.json`